src/base/bittorrent/session.cpp

```diff
@@ -441,6 +441,7 @@ Session::Session(QObject *parent)
 #if defined(Q_OS_WIN)
     , m_OSMemoryPriority(BITTORRENT_KEY("OSMemoryPriority"), OSMemoryPriority::BelowNormal)
 #endif
+    , m_isReannounceWhenAddressChangedEnabled(BITTORRENT_SESSION_KEY("ReannounceWhenAddressChanged"), false)
```

Comment:
Wrong order here, move it below `m_maxConcurrentHTTPAnnounces`.

Reply:
Fixed.